src/cogs/cli.py

```diff
@@ -22,13 +23,15 @@ def version(args):
 
 def main():
     parser = ArgumentParser()
+    global_parser = ArgumentParser(add_help=False)
+    global_parser.add_argument("-v", "--verbose", help="Print logging", action="store_true")
```

Comment:
Can `-v` just be added to `parser` and be inherited by all its subparsers, without defining a new `global_parser`?

Reply:
I tried that and it didn't work, so I searched for an answer and it looks like you have to create a separate "parent" - https://stackoverflow.com/questions/33645859/how-to-add-common-arguments-to-argparse-subcommands/33646419